docs/usage/cluster-interact.md

```diff
@@ -0,0 +1,31 @@
+# Interacting With your Cluster
+
+## OpenShift Client binary (oc)
+
+The `minishift start` command creates a "[oc context](http://kubernetes.io/docs/user-guide/kubectl/kubectl_config_set-context/)" called "minishift".
+This context contains the configuration to communicate with your minishift cluster.
+
+Minishift sets this context to default automatically, but if you need to switch back to it in the future, run:
+
+```
+oc config set-context minishift
+```
+
+or pass the context on each command like:
+```
+oc get pods --context=minishift`
+```
+
+## Console
+
```

Comment:
@Preeticp I think this is something that would need to be addressed later, there's an extensive technical review/update that is planned but it's OOS for this PR.